components/admin_console/system_roles/system_role/system_role_permissions.tsx

```diff
@@ -154,6 +161,11 @@ export default class SystemRolePermissions extends React.PureComponent<Props, St
             };
         }
 
+        if (!isLicensedForCloud) {
+            // Remove the billing section if it's not licensed for cloud
+            sectionsList.splice(1, 1);
```

Comment:
Could we filter here based on section name? It would be more readable and robust than relying on position in the case in the future someone adds another section before `billing` and alters the array positions.

Reply:
G000000D call on that :) Totally forgot about how in the future, the section may not have the same index. I added a findIndex for it.

Reply:
![THANKYOU](https://user-images.githubusercontent.com/17804942/100874752-0b46ba00-3473-11eb-8528-fc1857ff07e5.gif)